tests/api_connexion/endpoints/test_dataset_endpoint.py

```diff
@@ -53,21 +53,23 @@ class TestDatasetEndpoint:
     default_time = "2020-06-11T18:00:00+00:00"
 
     @pytest.fixture(autouse=True)
-    def setup_attrs(self, configured_app) -> None:
+    def setup(self, configured_app):
```

Comment:
`setup` actually has special meaning to Pytest in certain situations. That semantic is not in effect here (due to this being a fixture), but let’s not use the name anyway to avoid confusion.

Reply:
Gotcha, I actually took a quick peek for exactly that and came up empty. I've refactored it and I think this is better anyways. Thanks.